handlers/bin/command-not-found

```diff
@@ -91,18 +108,19 @@ try:
         rows = module.query_repo(repo, term)
 
     if rows == None or len(rows) == 0:
-        print >> sys.stderr, '\r', '%s:' % term, _('command not found'), lendel*' '
+        print('\r', '%s:' % term, _('command not found'),
+              lendel * ' ', file=sys.stderr)
         sys.exit(EX_NOTFOUND)
 
-    print >> sys.stderr, '\r', lendel*' ',
+    print('\r', lendel * ' ', end=' ', file=sys.stderr)
 
     for row in rows:
         if check_installed(term, row[1], row[2]):
             print_installed(term, row[1], row[2])
 
     print_found(rows)
 
-except:
-    pass
+except Exception as e:
+    raise e
```

Comment:
Silly question - what's the point of catching exceptions if you simply re-raise them?

(I'm not saying that `pass` aka silently ignoring errors was a good choice ;-)

Reply:
I think that you are right. I am simply porting the same behavior from Python 2 to Python 3, but this piece re-raise instead of pass, like old code. Maybe I did that to see some error while testing.

I will fix it!